src/octopeer-github/main/MainController.ts

```diff
@@ -71,17 +63,21 @@ class MainController {
      * @param tab   the Tab to check for
      */
     private testAndSend(tab: Tab) {
```

Comment:
That's been a while already. If you read the JavaDoc, you see what it does:
`Only sends a message to a tab if it contains the correct URL.`
I named it alike to a "test-and-set" operation that comes from concurrent programming.